src/indexing/transforms/labeling_pipeline.py

```
@@ -0,0 +1,336 @@
+        # Step 2: Extract section keywords (parallel)
+        section_keywords: Dict[str, SectionKeywords] = {}
+        if self.config.enable_keywords:
+            logger.info(
+                f"Step 2: Extracting keywords for {len(document.sections)} sections..."
+            )
+            tasks = [
+                self.keyword_extractor.extract_async(section, taxonomy.primary_category)
+                for section in document.sections
+            ]
+            results = await asyncio.gather(*tasks)
+            for section, keywords in zip(document.sections, results):
```

Ruff (0.14.6)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)